test/schedule/momo.integration.spec.ts

```diff
@@ -115,7 +114,7 @@ describe('Momo', () => {
 
     it('executes job that was executed before', async () => {
       const jobEntity = {
-        ...toJobDefinition(fromMomoJob(momoJob)),
+        ...toJobDefinition(toJob(momoJob)),
```

Comment:
Doesn't this change actually belong in the commit `refactor: change createJobEntity to toJobDefinition`, when we refactor `fromMomoJob` to `toJob`

Reply:
Yes, we could squash those commits, the refactoring commit and the fix commit that follows ... there were two broken imports (for some reason imported from dist instead src) so the renaming of the method didn't work and I only noticed that later

Reply:
Ah I think I understand now - we change this method in this commit because we previously imported `fromMomoJob` from `dist`. However, If I were to check out this commit and rebuild `dist`, then `fromMomoJob` would also not exist in `dist`. 

Personally, I might argue for squashing this together with the previous commit, but if you're ok with the previous commit being unable to run the integration tests, then we can leave it.

Reply:
Yes exactly

I think we once discussed that we are ok with having 'red' commits ... so I would just leave it as it is

Reply:
sounds good to me